updater_post_install.sh

```diff
@@ -64,6 +64,8 @@ hcaptcha_sitekey="$(grep hcaptcha.site_key /ebsa/config/services.config | cut -d
 hcaptcha_secret="$(grep hcaptcha.secret /ebsa/config/services.config | cut -d '=' -f 2 | head -1)"
 deepl_key="$(grep mastodon.deepl_key /ebsa/config/services.config | cut -d '=' -f 2 | head -1)"
 deepl_plan="$(grep mastodon.deepl_plan /ebsa/config/services.config | cut -d '=' -f 2 | head -1)"
+surf_registrations_enabled="$(grep mastodon.surf_registrations_enabled /ebsa/config/services.config | cut -d '=' -f 2 | head -1)"
+surf_registration_token="$(grep mastodon.surf_registration_token /ebsa/config/services.config | cut -d '=' -f 2 | head -1)"
```

Comment:
what happens if mastodon.surf_registration_token does not exist?  Will this work ok?

Reply:
If surf_registrations_enabled then we return a Forbidden if we're missing the token here.